tests/link/pytorch/test_extra_ops.py

```diff
@@ -41,3 +41,54 @@ def test_pytorch_CumOp(axis, dtype):
         out = pt.cumprod(a, axis=axis)
         fgraph = FunctionGraph([a], [out])
         compare_pytorch_and_py(fgraph, [test_value])
+
+
+@pytest.mark.parametrize("axis", [0, 1])
+def test_pytorch_Repeat(axis):
+    a = pt.matrix("a", dtype="float64")
+
+    test_value = np.arange(6, dtype="float64").reshape((3, 2))
+
+    out = pt.repeat(a, (1, 2, 3) if axis == 0 else (3, 3), axis=axis)
+    fgraph = FunctionGraph([a], [out])
+    compare_pytorch_and_py(fgraph, [test_value])
+
+
+@pytest.mark.parametrize("axis", [0, 1])
+def test_pytorch_Unique_axis(axis):
+    a = pt.matrix("a", dtype="float64")
+
+    test_value = np.array(
+        [[1.0, 1.0, 2.0], [1.0, 1.0, 2.0], [3.0, 3.0, 0.0]], dtype="float64"
+    )
+
+    out = pt.unique(a, axis=axis)
+    fgraph = FunctionGraph([a], [out])
+    compare_pytorch_and_py(fgraph, [test_value])
+
+
+@pytest.mark.parametrize(
+    "return_index, return_inverse, return_counts",
+    [
+        (False, True, False),
+        (False, True, True),
+        pytest.param(
+            True, False, False, marks=pytest.mark.xfail(raises=NotImplementedError)
+        ),
+    ],
+)
```

Comment:
Should these be distinict parametrizes to test all combinations?
```suggestion
@pytest.mark.parametrize("return_index", (False, pytest.param(True, marks=...)))
@pytest.mark.parametrize("return_inverse", (False, True))
@pytest.mark.parametrize("return_counts", (False, True))
```

Reply:
I didn't want to test all the combinations. `return_index=True` should always fail.  I wanna test that once instead of 4 times.  The combination `return_inverse=False` and `return_counts=False`  is tested somewhere else, etc.

Reply:
Any reason not to test all the combinations? How fast is the test running with 3 vs the 9?